conventions/build.gradle.kts

```diff
@@ -29,8 +28,7 @@ dependencies {
   implementation(gradleApi())
   implementation(localGroovy())
 
-  implementation("io.opentelemetry.instrumentation.muzzle-generation:io.opentelemetry.instrumentation.muzzle-generation.gradle.plugin")
-  implementation("io.opentelemetry.instrumentation.muzzle-check:io.opentelemetry.instrumentation.muzzle-check.gradle.plugin")
+  implementation("io.opentelemetry.instrumentation:gradle-plugins:1.9.1-alpha")
```

Comment:
These are the actual Maven coordinates, previously it just happened to work because we substitute anyways.

I also added a version here to make it easier to comment out `includeBuild`, even with it the version will generally not be used because of the substitution so maybe it doesn't hurt to have it here?